CHANGELOG.md

```diff
@@ -3,6 +3,51 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## [4.0.0-alpha.19](https://github.com/mui-org/material-ui-x/compare/v4.0.0-alpha.18...v4.0.0-alpha.19)
+
+###### _Feb 5, 2021_
+
+Big thanks to the 5 contributors who made this release possible. Here are some highlights ✨:
+
+- 🎁 Add getRowId prop (#972) @dtassone
+- 🚀 Add streaming delete row api (#980) @dtassone
+- 💅 Fix autoHeight (#940) @oliviertassinari
+- 🙌 Enable the data grid to work under strict mode (#933) @dtassone
+- ⚡️ Add component slots for toolbar and preference panel (#971) @DanailH
+- 🐛 Polish existing features, fix 9 issues.
+
+### @material-ui/x-grid@v4.0.0-alpha.19 / @material-ui/data-grid@v4.0.0-alpha.19
+
+#### Changes
```

Comment:
We could remove this subheader. It's really useful to separate breaking changes to the other ones.

Reply:
I missed it, there were no breaking changes.